DAL/BaseDao.cs

```diff
@@ -1,28 +1,21 @@
 ﻿using System.Configuration;
-using MongoDB.Bson.Serialization.Attributes;
-using MongoDB.Bson;
+using Model;
 using MongoDB.Driver;
 
 namespace DAL
 {
     public class BaseDao
     {
-        string connectionString;
-        MongoClient client;
+        protected readonly IMongoCollection<Employee> _employeeCollection;
+        protected readonly IMongoCollection<Ticket> _ticketCollection;
 
         public BaseDao()
         {
-            connectionString = ConfigurationManager.AppSettings["MongoDB"];
-            client = new MongoClient(connectionString);
-            
-            Test(client);
-        }
-
-        static async Task Test(MongoClient client)
-        {
-            // TEST
-            var database = client.GetDatabase("NoSQLCluster");
-            
+            var connectionString = ConfigurationManager.AppSettings["MongoDB"];
```

Comment:
Please, follow the official documentation of MongoDB and declare the connection string outside of the BaseDao constructor as a const string.
https://www.mongodb.com/docs/drivers/csharp/current/fundamentals/connection/connect/

```
const string connectionString = ConfigurationManager.AppSettings["MongoDB"];
```

Reply:
const doesn't work with the ConfigurationManager, so we will make it readonly.